kolibri/core/assets/src/vue/content-icon/index.vue

```diff
@@ -1,10 +1,12 @@
 <template>
 
-  <svg v-if="kind=='topic'" class="content-icon" src="./content-icons/topic.svg"></svg>
-  <svg v-if="kind=='video'" class="content-icon" src="./content-icons/video.svg"></svg>
-  <svg v-if="kind=='audio'" class="content-icon" src="./content-icons/audio.svg"></svg>
-  <svg v-if="kind=='document'" class="content-icon" src="./content-icons/document.svg"></svg>
-  <svg v-if="kind=='exercise'" class="content-icon" src="./content-icons/exercise.svg"></svg>
+  <div>
+    <svg v-if="kind=='topic'" class="content-icon" src="./content-icons/topic.svg"></svg>
```

Comment:
Make sense. I think it is fine to leave as is because it would take more many more lines of code than we are trying to save.